packages/FlexGrid/Col/Col.jsx

```diff
@@ -5,42 +5,65 @@ import { Subscriber } from 'react-broadcast'
 import { Col as ReactFlexboxGridCol } from 'react-flexbox-grid'
 
 import safeRest from '../../../shared/utils/safeRest'
+import { deprecate } from '../../../shared/utils/warn'
 import styles from './Col.modules.scss'
 
-const removeProps = ({
-  xs,
-  sm,
-  md,
-  lg,
-  xl,
-  xsOffset,
-  smOffset,
-  mdOffset,
-  lgOffset,
-  xlOffset,
-  ...rest
-}) => safeRest(rest)
-
-const Col = ({ span, offset, children, ...rest }) => (
-  <Subscriber channel="flex-grid">
-    {gutter => (
-      <ReactFlexboxGridCol
-        {...removeProps(rest)}
-        xs={span || true}
-        xsOffset={offset}
-        className={gutter ? styles.padding : styles.gutterless}
-      >
-        {children}
-      </ReactFlexboxGridCol>
-    )}
-  </Subscriber>
-)
+const removeProps = ({ xsOffset, smOffset, mdOffset, lgOffset, xlOffset, ...rest }) =>
```

Comment:
Would we still want the offsets so that the API is more similar to the react-flexbox-grid component? https://github.com/roylee0704/react-flexbox-grid/blob/master/src/components/Col.js#L13-L17

Reply:
We could, but those were not listed in the ACs, so I feel like they might be out of scope.